backend/cloudwatch.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
-	"github.com/buildkite/buildkite-metrics/collector"
+	"github.com/buildkite/buildkite-agent-metrics/collector"
```

Comment:
I'd pull out all the renames of `github.com/buildkite/buildkite-metrics` to `github.com/buildkite/buildkite-agent-metrics` into a separate PR.

Reply:
I've created https://github.com/buildkite/buildkite-agent-metrics/pull/56

Reply:
https://github.com/buildkite/buildkite-agent-metrics/pull/56